src/main/scala/ReproducibleBuildsPlugin.scala

```diff
@@ -101,12 +95,62 @@ object ReproducibleBuildsPlugin extends AutoPlugin {
       reproducibleBuildsPackageName.value,
       ext,
       ext,
-      "compile",
+      Compile.name,
       extraModuleAttributes,
       null
     )
   }
 
+  def locationFromResolver(r: DependencyResolver, ext:String) = Def.task {
+    import SbtLibraryManagementFunctions._
+    val moduleInfo = (moduleSettings.value match {
+      case ic: InlineConfiguration =>
+        val icWithCross = substituteCross(ic)
+        if (sbtPlugin.value) appendSbtCrossVersion(icWithCross)
+        else icWithCross
+    }).moduleInfo
+
+    val artifactOrigin = r.locate(
+      toIvyArtifact(
+        newConfiguredModuleID(moduleID.value, moduleInfo, Vector(Compile)),
+        artifact.value.withExtension(ext),
```

Comment:
The one thing that I am unsure here is whether using the sbt `artifact` key here is correct, i.e. is it being correctly propagated in this context (as I said in https://github.com/raboof/sbt-reproducible-builds/pull/254/files#r1276248807 this isn't really tested).